board/drivers/bootkick.h

```diff
@@ -0,0 +1,54 @@
+bool bootkick_ign_prev = false;
+BootState boot_state = BOOT_BOOTKICK;
+uint8_t bootkick_harness_status_prev = HARNESS_STATUS_NC;
+
+uint8_t boot_reset_countdown = 0;
+uint8_t waiting_to_boot_countdown = 0;
+bool bootkick_reset_triggered = false;
+
```

Comment:
Might want to make a reset function for the various state variables. For example if the device sends a heartbeat or receives another ignition before `boot_reset_countdown` is 0, it will leave the reset state only to enter it again again on the next loop.